app/routes/routes.py

```diff
@@ -0,0 +1,82 @@
+
+from os import abort
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+
+@planets_bp.route("", methods=["POST"])
+def handle_planets():  
```

Comment:
Just curious, why name the function `handle_planets` instead of `create_planet`?